examples/MinimalModemGPSExample/MinimalModemGPSExample.ino

```diff
@@ -200,13 +201,10 @@ void loop()
 
         // After successful positioning, the PMU charging indicator flashes quickly
         PMU.setChargingLedMode(XPOWERS_CHG_LED_BLINK_4HZ);
-        while (1) {
-            delay(1000);
-        }
     } else {
         // Blinking PMU charging indicator
         PMU.setChargingLedMode(level ? XPOWERS_CHG_LED_ON : XPOWERS_CHG_LED_OFF);
         level ^= 1;
-        delay(1000);
     }
+    delay(1000);
```

Comment:
We are waiting for 1 second regardless if we got GPS coordinates or not yet. So moving this `delay` call outside of the `if`.